finder-app/Makefile

```diff
@@ -0,0 +1,33 @@
+
+TARGET_EXEC := writer
+
+BUILD_DIR := ./build
+SRC_DIR := ./
+
+SRCS := $(wildcard $(SRC_DIR)/*.c)	# Find all c files
+OBJS := $(SRCS:$(SRC_DIR)/%.c=$(BUILD_DIR)/%.o)	# make object files for all c files
+
+CC = gcc
+CFLAGS = -g
+
+CROSS_COMPILE ?=
+
+ifneq ($(CROSS_COMPILE),)
```

Comment:
@jackcenter there's a simpler way to handle cross compile, we'll discuss in class tomorrow.